packages/ui-options/src/Options/Item/index.tsx

```diff
@@ -72,8 +73,12 @@ class Item extends Component<OptionsItemProps> {
     this.props.makeStyles?.()
   }
 
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'renderLabel' implicitly has an 'any' ty... Remove this comment to see the full error message
-  renderContent(renderLabel, contentVariant) {
+  renderContent(
+    renderLabel: OptionsItemProps['renderBeforeLabel'],
```

Comment:
The types are the same, but maybe we could indicate that it is either this or the After version
```suggestion
    renderLabel: OptionsItemProps['renderBeforeLabel'] | OptionsItemProps['renderAfterLabel'],
```